README.md

```diff
@@ -1,2 +1,52 @@
-# vc-powershell-template
-Powershell script for creating Virto Commerce module solution
+# VC module template
+
+## Overview
+
+This script is a lite version of [Virto Commerce 2.x Module Templates](https://marketplace.visualstudio.com/items?itemName=Virto-Commerce.VirtoCommerceModuleTemplates)
+The script creates a solution template for the VirtoCommerce platform new module. The script automatically creates all the necessary projects:
+
+* Core - Business and Application Models
+* Data - Data Access and Implementation of business logic
+* Tests - xUnit Unit tests
+* Web - API Controllers
+
+and names the folders in accordance with the rules used in the VirtoCommerce development team.
```

Comment:
used by the VirtoCommerce development team.

Reply:
fixed